src/Agent/NewRelic/Agent/Core/Utilities/ExtensionsLoader.cs

```diff
@@ -61,7 +61,8 @@ public static void Initialize(string installPathExtensionsDirectory)
 
                 { "OpenConnectionTracer",                                                                           Path.Combine(_installPathExtensionsDirectory, "NewRelic.Providers.Wrapper.Sql.dll") },
                 { "OpenConnectionWrapper",                                                                          Path.Combine(_installPathExtensionsDirectory, "NewRelic.Providers.Wrapper.Sql.dll") },
-
+                { "SerilogCreateLoggerWrapper",                                                                          Path.Combine(_installPathExtensionsDirectory, "NewRelic.Providers.Wrapper.SerilogLogging.dll") },
```

Comment:
I think it might be helpful to have a comment here to explain why Serilog is the only logging framework that is on this list